scripts/submitHistoFiller_lxplus.py

```diff
@@ -0,0 +1,147 @@
+import os
+import sys
+import argparse
+
+
+write = lambda stream, text: stream.write(text + '\n')
+
+
+def make_parser():
+    parser = argparse.ArgumentParser(description='Command line parser of plotting options')
+    parser.add_argument('--outdir', type=str, dest="outdir", required=True,
```

Comment:
`outdir` is a somewhat confusing name for this argument. Could you change it to `submitdir` or something similar?

Reply:
outdir is now called submit_dir.